frontend/packages/operator-lifecycle-manager/src/components/clusterserviceversion.tsx

```diff
@@ -768,6 +769,7 @@ export const ClusterServiceVersionDetails: React.SFC<ClusterServiceVersionDetail
   const {
     'marketplace.openshift.io/support-workflow': marketplaceSupportWorkflow,
     'olm.targetNamespaces': olmTargetNamespaces = '',
+    'operatorframework.io/initialization-resource': crInitializationResource,
```

Comment:
nit: I'd just call this `initializationResource`. It matches the annotation and whether it's a CR is unimportant.

```suggestion
    'operatorframework.io/initialization-resource': initializationResource,
```